PSCompatibilityCollector/Tests/UtilityApi.Tests.ps1

```diff
@@ -155,9 +139,16 @@ Describe "PowerShell version object" {
             $v.Patch | Should -Be $Patch
             $v.PreReleaseLabel | Should -BeExactly $Label
             $v.BuildLabel | Should -Be $BuildLabel
-        }
+        } -TestCases @(
```

Comment:
This is an interesting version of the inline `-TestCases` parameter, vs the one I've seen in the PowerShell repo where the cases come before the test (like with xUnit theories)

Reply:
Moved `-Testcases` before `-Test` in 932ca1033b905513d775215c6f8880adecd2ff79